### PR TITLE
feat: allow running timer in the caller's process

### PR DESCRIPTION
**Pull request by spacewander**
And change ngx.timer.at to ngx.timer.every

### CODE REVIEW

**Comment by wu-sheng** [comment]
Could you explain the pull request?

**Comment by spacewander** [comment]
Currently the client is forced to run in the worker 0. However, in some cases we may want the client to be run in a specific worker process decided by the caller.

File: lib/skywalking/client.lua [hunk 1]
```diff
@@ -21,12 +21,12 @@ local Client = {}
 
 -- Tracing timer reports instance properties report, keeps alive and sends traces
 -- After report instance properties successfully, it sends keep alive packages.
-function Client:startBackendTimer(backend_http_uri)
+function Client:startBackendTimer(backend_http_uri, in_current_process)
```
Comment (wu-sheng) [inline lib/skywalking/client.lua:24]: How to set this? I remember Nginx shares the same configuration across different processes. How to make them using different?

Comment (spacewander) [reply lib/skywalking/client.lua:24]: @wu-sheng 
They are some ways in Nginx to communicate among processes, like using the shared memory, listening different port in different process, etc. And there is different ID to different worker process. So different Nginx process can be assigned different job.

Comment (wu-sheng) [reply lib/skywalking/client.lua:24]: Yes, I know that question. My question is, this is a user API parameter, how a user is going to set this? Or what way do you expect to set?
There is no doc, use case, or example updated, I can't get the idea of adding this.

**Comment by spacewander** [comment]
@wu-sheng 
I have changed my mind. I don't need this feature, so I will close it. Thanks for your review.

**Pull request closed without merging**